app/controllers/tasks_controller.rb

```diff
@@ -18,6 +18,8 @@ def show
   def create
     @task = Task.new(task_params)
 
+    @task.author = current_user
```

Comment:
I was used to it before params white lists were implemented :innocent: 

I am changing it